R/constants.R

```diff
@@ -45,9 +54,12 @@ CONV_BIL_MIL <- 1000
 CONV_MIL_THOUS <- 1000
 CONV_ONES_THOUS <- 0.001
 CONV_TON_MEGATON <- 1e-6
+CONV_T_KG <- 1e3
+CONV_T_METRIC_SHORT <- 1000/908
 CONV_MCAL_PCAL <- 1e-9
 CONV_HA_BM2 <- 1e-5
 CONV_1990_2010_USD <- 1.510
+CONV_2004_1975_USD <- 0.3472
```

Comment:
@bpbond: I think we are going to have a lot of these (per old data system). Do we want to come up with a better system for this?